### ISSUE TITLE
Use `ap` instead of `activitypub`

### ISSUE THREAD

**Issue by BANOnotIT** [activitypub-schema/proposal#2]
It might be efficient to use shorter version of URI schema.

**Comment by almereyda** [comment]
Why not just `pub`?

**Comment by BANOnotIT** [comment]
`ap` is smaller and corresponds to ActivityPub more then a general `pub`

**Comment by almereyda** [comment]
Indeed. It would also follow the naming convention for how these protocols are translated into protocol schemes:

- File Transfer Protocol ⯈ FTP
- HyperText Transfer Protocol ⯈ HTTP
- Simple Mail Transfer Protocol ⯈ SMTP
- ...

In all of which an abbreviation of the actual name of the implementation is used for the schema.
